src/arco_era5/resize_zarr.py

```diff
@@ -147,3 +147,33 @@ def resize_zarr_target(target_store: str, end_date: datetime, init_date: str,
         logger.info(f"Consolidation of {target_store} is completed.")
     else:
         logger.info(f"Data is already resized for {target_store}.")
+
+
+def update_zarr_metadata(url: str, time_end: datetime.date, metadata_key: str = '.zmetadata') -> None:
+    try:
+        attrs = {"valid_time_start": "1940-01-01",
+                 "valid_time_end": str(time_end),
+                 "last_updated": str(datetime.datetime.now())
```

Comment:
By default, datetime.now() uses local timezone on the machine running this code.

Instead let's specify UTC:
```suggestion
                 "last_updated": str(datetime.datetime.utcnow())
```

Reply:
updated.